micronota/bfillings/infernal.py

```diff
@@ -0,0 +1,154 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, micronota development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from os import remove, close
+from tempfile import mkstemp
+
+from skbio import DNA, RNA
+from burrito.parameters import FlagParameter, ValuedParameter
+from burrito.util import CommandLineApplication, ResultPath
+
+from .util import _get_parameter
+from .model import ModelFetch, ModelPress, ModelScan
+
+class CMScan(ModelScan):
+    '''cmscan application controller.
+
+    cmscan is used to search a sequence against a covariance model database.
+    This wrapper is tested for INFERNAL 1.1.1 (July 2014)
+    '''
+    _command = "cmscan"
+    _valued_path_options = ModelScan._valued_path_options
```

Comment:
Ok, this is where things get fuzzy.

So _value_path_options is not actually an instance variable - it is a class variable.
Class variables as it turns out are not implicitly inherited - they need to be explicitly inherited.

Just as a sanity check, I ran this by commenting out `_valued_path_options`, and it fails as expected